wgs/cnai/glossary/index.md

```diff
@@ -9,6 +9,18 @@ Defines key terms used in the Cloud Native Working Group’s writings.
 
 See also: [CNCF Glossary](https://glossary.cncf.io/)
 
+## AI for CN
+
+A class of tools integrates AI with the intent to optimize CN system management outcomes. Optimization can take different forms, from actionable advice to agentic actions.
+
+## CN for AI
```

Comment:
Should we use the full word "Cloud Native  for AI" or "Cloud Native (CN) for AI"